docs/Complex-Number/index.md

```diff
@@ -0,0 +1,76 @@
+# Average
```

Comment:
The documentation is for **Complex Numbers**!

Reply:
Oh I'm sorry. I will resolve the issues right now